packages/deno-npm-prefix/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@nodejs-loaders/deno-npm-prefix",
+  "description": "NPM protocol loader for Node.js",
```

Comment:
```suggestion
  "description": "'npm:' prefix loader for Node.js",
```

Reply:
what about

> Support 'npm:' prefix on node via customization hooks

Reply:
Let's go with the consensus from https://github.com/nodejs-loaders/nodejs-loaders/issues/69 🙂